lua/mason-registry/docker-compose-language-service/init.lua

```diff
@@ -0,0 +1,12 @@
+local Pkg = require "mason-core.package"
+local npm = require "mason-core.managers.npm"
+
+return Pkg.new {
+    name = "docker-compose-language-service",
+    desc = [[A language server for Docker Compose.]],
+    homepage = "https://github.com/microsoft/compose-language-service",
+    languages = { Pkg.Lang.YAML },
+    categories = { Pkg.Cat.LSP },
+    install = npm.packages { "@microsoft/compose-language-service" },
```

Comment:
They don't seem to be providing an executable to launch the server. I think we should provide one via Mason - or is there some reason there's no executable provided?

Reply:
You are right, they don't provide it.

I figured I started with just the installation and try adding the executable afterwards.

However, I am struggling to find a nice way of doing so :S